website/pages/docs/nia/tasks.mdx

```diff
@@ -8,4 +8,24 @@ description: >-
 
 # Tasks
 
-A task is the translation of dynamic service information from the Consul Catalog into network infrastructure changes downstream. Consul-Terraform-Sync carries out automation for executing tasks using network drivers. The scope of a task in terms of the Terraform driver is the execution of a Terraform module. An example of a task is to automate a firewall security policy rule with discovered IP addresses for a set of Consul services.
+A task is the translation of dynamic service information from the Consul Catalog into network infrastructure changes downstream. Consul-Terraform-Sync carries out automation for executing tasks using network drivers. For a Terraform driver, the scope of a task is a Terraform module.
+
+Below is an example task configuration:
+
+```shell-session
+task {
+  name = "frontend-firewall-policies"
+  description = "Add firewall policy rules for frontend services"
+  providers = ["fake-firewall", "null"]
+  services = ["web", "image"]
+  source = "example/firewall-policy/module"
+  version = "1.0.0"
+}
+
+```
+
+In the example task above, the "fake-firewall" and "null" providers, listed in the `providers` field, are used. These providers themselves should be configured in their own separate [provider blocks](/docs/nia/installation/configuration#provider). These providers are used in the Terraform module "example/firewll-policy/module", configured in the `source` field, to create, update, and destroy resources. This module may do something like use the providers to create and destroy firewall policy objects based on IP addresses. The IP addresses come from the "web" and "image" service instances configured in the `services` field. This service-level information is retrieved by Consul-Terraform-Sync which watches Consul catalog for changes.
```

Comment:
I believe the casing can be one of "Consul Catalog", "Consul's catalog", or "the catalog"

Reply:
👌 went with "Consul Catalog". thanks for the feedback!